### PR TITLE
02_Strain_Energy_Minimization_3D_Shell

### PR DESCRIPTION
**Pull request by ge73xip**
Requesting review

### CODE REVIEW

**Comment by rubenzorrilla** [comment]
Just mentioning that it would be nice that at some point the shape opt. examples follow the use_cases and validation structure of the rest of the repo.

**Comment by armingeiser** [comment]
Looks good in general. Two comments:

- Pease use png instead of svg format for everything that is not a vector graphic. (i.e. the setup figure)
- Please add a simple graph of the objective function in the results section.

**Comment by armingeiser** [comment]
> Just mentioning that it would be nice that at some point the shape opt. examples follow the use_cases and validation structure of the rest of the repo.

Can do - everything will be use_case though. @ge73xip please make a PR creating the same folder structure as for the other apps (once we have merged the currently open PRs)

**Comment by rubenzorrilla** [comment]
> > Just mentioning that it would be nice that at some point the shape opt. examples follow the use_cases and validation structure of the rest of the repo.
> 
> Can do - everything will be use_case though. @ge73xip please make a PR creating the same folder structure as for the other apps (once we have merged the currently open PRs)

Thanks guys!

**Comment by ge73xip** [comment]
> Looks good in general. Two comments:
> 
> * Pease use png instead of svg format for everything that is not a vector graphic. (i.e. the setup figure)
> * Please add a simple graph of the objective function in the results section.

changes made accordingly

**Review by armingeiser — COMMENTED** [review]
Please delete the images that are not used - then you can merge.

File: shape_optimization/.gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 *.vtk
 *.bin
 *.post.lst
+*.ipynb
```
Comment (armingeiser) [inline shape_optimization/.gitignore:6]:
```suggestion
```

**Pull request merged**